### DISCUSSION TITLE
购买了专业版，软件更新后显示免费版，尝试邮箱登录，邮箱收不到恢复登录验证码，如何解决？我的邮箱是852006523@qq.com

### DISCUSSION THREAD

**Discussion by RXDK** [getlantern/lantern-forum-cn#429] [Q&A]
![6025105ad97e71f6f37eb0c011fe772](https://github.com/getlantern/lantern-forum-cn/assets/166693009/96f7ee47-013a-4b19-b0af-fad541a45222)

**Comment by MaxMKMax** [comment]
@RXDK 好的. 我已经帮您提交问题上去处理了. 请您留意邮箱或GitHub. 谢谢您的等待!

**Reply by RXDK** [reply]
好的，谢谢

**Reply by MaxMKMax** [reply]
@RXDK 不客气. 谢谢您支持蓝灯! 过几天请查看邮箱或GitHub. 谢谢.
